packages/server/src/service/mg.ts

```diff
@@ -85,3 +86,35 @@ export const sendSignupEmail = async (
   const emailData = toSignupEmailData(letter, voterEmail, officialEmails, { pdfBuffer, force })
   return mg.messages().send(emailData)
 }
+
+const mailgunToGCPLogLevels = {
+  "info": "INFO",
+  "warn": "WARNING",
+  "temporary": "WARNING",
+  "error": "ERROR"
+}
+export const logMailgunLogToGCP = async (request: Object): Promise<void> => {
+  // if (request.body == null) {
+  //   return
+  // }
+
+  const logging = new Logging({projectId: processEnvOrThrow('GCLOUD_PROJECT')})
+  const log = logging.log('mailgun-log')
+  // if (request.body.log_level instanceof string) {
+  //   throw new Error('test')
+  // }
+  // const logLevel: string = mailgunToGCPLogLevels[requestBody.log_level]
```

Comment:
Always get errors about index needing to be of type string but could by of type any.  What's the right way to handle this?